internal/syncer/connector/confluence.go

```
@@ -0,0 +1,885 @@
+	}
+	req.Header.Set("Accept", "application/json")
+	if c.isCloud || c.username != "" {
+		req.SetBasicAuth(c.username, c.accessToken)
+	} else {
+		req.Header.Set("Authorization", "Bearer "+c.accessToken)
+	}
+	res, err := c.client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer res.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `res.Body.Close` is not checked

(errcheck)